### PR TITLE
Week 15, Project-todos, Therese Hagelin

### PR DESCRIPTION
**Pull request by ThereseHag**
Link to my deployed project:
https://todo-app-by-therese-hagelin.netlify.app/

My reflections on how this week's project turned out:

This week was very fun. I really enjoyed getting more time to work with Redux, and I enjoy it a lot. I will need some time to digest mutability and immutability, but all in all think this week went really good. 

Things I'd like to have clarified or explained in more detail:

### CODE REVIEW

File: code/src/App.js [hunk 1]
```diff
@@ -1,9 +1,29 @@
...
 
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
-  )
-}
+    <Provider store={store}>
+      <div className="main-content">
```
Comment (Irina-web-dev) [inline code/src/App.js:21]: Clever! 🙌 I was trying to figure out where to add main container and ended up by adding it in the index.html file 🤷

File: code/src/App.js [hunk 2]
```diff
@@ -1,9 +1,29 @@
...
+        <Header />
+        <TodoForm />
+        <TodoList />
+        <Footer />
```
Comment (Irina-web-dev) [inline code/src/App.js:25]: With these components´ names it is easy to understand a responsibility of each component 👍

File: code/src/components/Todolist.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+          <input
+            className="description"
+            type="checkbox"
+            checked={todo.isComplete}
+            onChange={() => dispatch(todos.actions.toggleComplete(todo.id))}
+          />
+          <label htmlFor="description">
+            <p>{todo.description}</p>
+          </label>
+          <span className="date"> ✯ - {moment(todo.createdAt).fromNow()}</span>
```
Comment (Irina-web-dev) [inline code/src/components/Todolist.js:45]: I like this timestamp ✨

**Review by Irina-web-dev — APPROVED** [review]
Hi Therese! I really like the design of your todos app, I would love to use this app myself :) Maybe one suggestion is to apply `justify-content : space-between ` for todo list (see pic). Otherwise list is not even on sides.

Your code is clean, easy to follow and according to guidelines. All I can say is great job, be proud and celebrate with a glass of red wine 🍷

Capture

File: code/README.md [hunk 4]
```diff
@@ -1,19 +1,18 @@
...
 
-1. Download & extract the [latest release](https://github.com/Technigo/react-starter/releases/latest) from GitHub
-1. Open the directory in the terminal: `cd /path/to/react-starter`
-1. Install required dependencies with npm: `npm install`
-1. Start the project: `npm start`
+## View it live
+
+The to do app can be seen here:
+
+https://todo-app-by-therese-hagelin.netlify.app/
```
Comment (KarolCislowski) [inline code/README.md:18]: doubled ReadMe file?? XeroX Attacks!!! 👽 👽

File: code/src/index.css [hunk 5]
```diff
@@ -11,3 +17,150 @@ code {
...
+  background: #8f8681;
+  display: flex;
+  align-self: center;
+  align-items: center;
+  justify-content: center;
+  padding: 40px;
+}
+
+.input-field {
+  background-color: rgb(225, 220, 217);
```
Comment (KarolCislowski) [inline code/src/index.css:79]: please decide if you are using rgb or hex color notation

**Review by KarolCislowski — APPROVED** [review]
Great app 🦄

**Pull request closed without merging**